app/controllers/api/articles_controller.rb

```diff
@@ -22,7 +22,13 @@ def index
 
   def show
     article = Article.find(params[:id])
-    if article.published?
+    if request.headers[:source] == 'admin-system'
+      if article_evaluation(article)
+        render json: article, serializer: ArticlesShowSerializer
```

Comment:
If we do this logic here, we should do it in the Update action as well

Reply:
Fixed